src/intersection-observer.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'intersection-observer';
```

Comment:
Honestly we can probably remove the polyfill for intersection observer since it has pretty broad browser support nowadays. Happy to do this in a followup PR though.

Reply:
yeah will do so in a follow up!